Gitlab/two-stage-pipeline-template/{{cookiecutter.outputDir}}/assume-role.sh

```diff
@@ -0,0 +1,18 @@
+REGION=$1
+ROLE=$2
+SESSION_NAME=$3
+
+unset AWS_SESSION_TOKEN
+export AWS_DEFAULT_REGION=$REGION
+
+export AWS_ACCESS_KEY_ID=$(echo $AWS_ACCESS_KEY_ID_)
+export AWS_SECRET_ACCESS_KEY=$(echo $AWS_SECRET_ACCESS_KEY_)
+
+cred=$(aws sts assume-role --role-arn "$ROLE" \
+                           --role-session-name "$SESSION_NAME" \
+                           --query '[Credentials.AccessKeyId,Credentials.SecretAccessKey,Credentials.SessionToken]' \
+                           --output text)
+
+export AWS_ACCESS_KEY_ID=$(echo $cred | awk '{ print $1 }')
+export AWS_SECRET_ACCESS_KEY=$(echo $cred | awk '{ print $2 }')
+export AWS_SESSION_TOKEN=$(echo $cred | awk '{ print $3 }')
```

Comment:
The shellcheck suggests using `"` to replace `'` in this script and then awk just stops working... gotta keep single quotes in this script.

Reply:
Are we sure that AccessKeyId, SecretAccessKey and SessionToken can't contain whitespaces? otherwise these values will be invalid

Reply:
I believe they can't have whitespaces. Also I saw this demo: https://gitlab.com/gitlab-examples/aws-sam/-/blob/master/assume-role.sh using the same way.

Reply:
> Are we sure that AccessKeyId, SecretAccessKey and SessionToken can't contain whitespaces? otherwise these values will be invalid

Although I never see spaces in them, I tried to look up in the doc it is never mentioned.

Reply:
[update] I don't it is an issue @elbayaaa 

```
~/P/a/t/i/pipeline ❯❯❯ export HEY=$(echo 'I have space')
~/P/a/t/i/pipeline ❯❯❯ echo $HEY
I have space
```